### ISSUE TITLE
Name "Data::Dumper::Indent" used only once

### ISSUE THREAD

**Issue by ggzengel** [alaskacommunications/nagios_check_drbd9#2]
I get this error:
> Name "Data::Dumper::Indent" used only once: possible typo at /var/lib/icinga2/checks/check_drbd9 line 45.

**Comment by ggzengel** [comment]
I'm not so familiar with perl and googled for a resolution.
I have found two solutions, but don't know if they are right:

```
use Data::Dumper;
{
  local $Data::Dumper::Indent = 1;
}
```

or

```
$Data::Dumper::Indent = 1;
$Data::Dumper::Indent = $Data::Dumper::Indent;

**Comment by syzdek** [comment]
Resolved this issue in commit 976e7fef803d7e1acdf7b240c6e173f2f161961f by removing unused references to Data::Dumper.
